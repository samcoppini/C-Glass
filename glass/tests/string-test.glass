{M
    [m
        (_a)A!
        (_o)(Output)!
        (_s)(String)!

        "Testing String.cmp:"(_o)(ol).?

        "hey""hey"(_s)(cmp).?<0>(_a)e.?(_o)(onl).?
        "heyy""hey"(_s)(cmp).?<0>(_a)(gt).?(_o)(onl).?
        "hey""heyy"(_s)(cmp).?<0>(_a)(lt).?(_o)(onl).?
        "abc""bcde"(_s)(cmp).?<0>(_a)(lt).?(_o)(onl).?
        "vbc""bcde"(_s)(cmp).?<0>(_a)(gt).?(_o)(onl).?
    ]
}
