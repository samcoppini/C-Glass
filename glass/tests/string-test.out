Testing String.cmp:
1
1
1
1
1
